styles/variables.less

```
@@ -625,7 +625,7 @@
 // ---------------------------------------
 @sand-input-popup-component-v-spacing: 42px;
 @sand-input-numbercell-border-radius: 12px;
-@sand-input-fullscreen-padding: (210px - .extract(@sand-popup-padding, top)[]) (480px - .extract(@sand-popup-padding, right)[]) (246px - .extract(@sand-popup-padding, bottom)[]) (480px - .extract(@sand-popup-padding, left)[]);
+@sand-input-fullscreen-padding: (210px - .extract(@sand-popup-padding, top)[]) 0 (246px - .extract(@sand-popup-padding, bottom)[]);
```

We do have GUI guidelines to have this padding actually.
We could reduce the number but it can't be 0.
I'll talk with the person in charge and let you know.

---

GUI person confirmed that below is the right value.
Because we have a back button with a shadow effect top left, we should keep the left and right padding.
The title area seems a little small but I think it's better than now.

```suggestion
@sand-input-fullscreen-padding: (210px - .extract(@sand-popup-padding, top)[]) (264px - .extract(@sand-popup-padding, right)[]) (246px - .extract(@sand-popup-padding, bottom)[]) (264px - .extract(@sand-popup-padding, left)[]]);
```